report_thesis/src/sections/introduction.tex

```diff
@@ -0,0 +1,45 @@
+\section{Introduction}\label{sec:introduction}
+The NASA Viking missions in the 1970s were the first to successfully land a rover on Mars, aiming to determine if life existed on the planet.
+Despite one experiment suggesting the presence of life, the results were ambiguous and inconclusive, and NASA was unable to repeat the experiment due to budget constraints\cite{marsnasagov_vikings}.
+
+A few decades later, the philosophy of Martian exploration had shifted from searching for life to investigating whether Mars ever had the conditions to support life as we know it.
+The \gls{mer} mission, which included the Spirit and Opportunity rovers, discovered clear evidence that water once flowed on Mars.
+As water alone is not enough to support life, NASA then shifted their focus to the search for organic material as well\cite{marsnasagov_observer, marsnasagov_spirit_opportunity}.
+
+The Curiosity rover landed on Mars in August 2012 inside Gale Crater as part of the \gls{msl} mission with this very purpose.
+Its sophisticated equipment quickly discovered that the conditions to support life as we know it had existed on Mars through chemical and mineral evidence.\cite{marsnasagov_chemcam}
+
+One of the instruments aboard the rover is the \gls{chemcam} instrument, which is a remote-sensing laser instrument used to gather \gls{libs} data from geological samples on Mars.
+\gls{libs} is a non-invasive technique that enables rapid analysis without the need for sample preparation by using a laser to ablate and remove surface contaminants to expose the underlying material and generate a plasma plume from the now-exposed sample material.
+This plasma plume emits light that is captured through three distinct spectrometers to collect a series of spectral readings.
+These spectra consist of various emission lines, each associated with a specific element, and their intensity reflects the concentration of that element in the sample.
+Consequently, a spectra serves as a complex, multi-dimensional fingerprint of the elemental composition of the examined geological formations.\cite{cleggRecalibrationMarsScience2017}
+
+Analyzing \gls{libs} data is computationally challenging due to high multicollinearity within spectral data, which diminishes the effectiveness of traditional linear analysis.
+The multicollinearity, which stems from correlations among spectral channels and elemental emission characteristics, complicates data interpretation.
+Additionally, there are multiple interacting physical processes
```

Comment:
Unfinished sentence or missing a full stop?

Reply:
Fixed earlier if you refer to `Additionally, there are multiple interacting physical processes`